Sources/PipeableSDK/ScriptContext.swift

```diff
@@ -0,0 +1,25 @@
+import JavaScriptCore
+
+@objc protocol ScriptContextJSExport: JSExport {
+    func scriptStarted()
+    func scriptEnded()
+}
+
+final class ScriptContext: NSObject, ScriptContextJSExport {
+    let dispatchGroup: DispatchGroup
+
+    init(_ dispatchGroup: DispatchGroup) {
+        self.dispatchGroup = dispatchGroup
+        super.init()
+    }
+
+    func scriptStarted() {
+        dispatchGroup.enter()
+    }
+
+    func scriptEnded() {
+        Task {
+            self.dispatchGroup.leave()
```

Comment:
Why does this need to be run in a Task detached from the executing thread? Do we need to post these on a queue or this kind of fire & forget works?

Reply:
Good question... To be honest, there should be no problem running it without a Task...

Reply:
I will try it and remove it if I don't find out why it needs to be like that :)